app.js

```diff
@@ -503,11 +503,11 @@ function setupModelRoutes(mongoConnection) {
 
 		res.status(err.status || 500);
 
-		if (req.xhr)
-			return res.json({ message: err.message });
+		if (req.xhr || (req.query.ajax === '1'))
```

Comment:
@kschzt  i've added this so that we can explicitly request json responses from backend. i can take it out but it makes my own life miserable when double-checking responses manually from the console.